torch/onnx/utils.py

```diff
@@ -44,7 +44,7 @@ def set_training(model, mode):
 
 def export(model, args, f, export_params=True, verbose=False, training=False,
            input_names=None, output_names=None, aten=False, export_raw_ir=False,
-           operator_export_type=None, opset_version=None):
+           operator_export_type=None, opset_version=None, reserve_param_name=False):
```

Comment:
A minor point - could we consider another name for this parameter, such as `retain_param_name` or `export_with_param_name`?